app/models/folio/bib_data.rb

```diff
@@ -0,0 +1,71 @@
+# frozen_string_literal: true
+
+module Folio
+  # Bibliographic data from Folio.
+  class BibData
+    def initialize(request, _ = nil)
+      @request = request
+    end
+
+    attr_reader :request
+
+    def title
```

Comment:
We should probably verify that edits to the MARC bib record sync to these FOLIO fields

Reply:
I have no idea how any of that works. 🤷

Reply:
Alissa states that "an edit to the 24x in SRS will trigger and automatic update to the instance."